src/pybkgmodel/data.py

```diff
@@ -654,25 +662,20 @@ def load_events(cls, file_name):
                             event_data[name] *= u.one
 
                 # Event times need to be converted from Instrument reference epoch
-                ref_epoch = astropy.time.Time(evt_head['MJDREFI']+evt_head['MJDREFF'], format='mjd')
+                ref_epoch = astropy.time.Time(evt_head['MJDREFI'], evt_head['MJDREFF'], format='mjd')
 
-                event_data['mjd'] = astropy.time.Time((evt_data['TIME'].to_numpy() 
-                                                       + ref_epoch.unix),
-                                                      scale='utc',
-                                                      format='unix'
+                event_data['mjd'] = astropy.time.Time((evt_data['TIME'].quantity 
+                                                       + ref_epoch)
                                                       ).mjd * u.d
 
-                # Adapt
-                evt_time = astropy.time.Time(event_data['mjd'], format='mjd')
-
                 # TODO: current observatory location only La Palma, no mandatory header keyword
                 obs_loc  = EarthLocation(lat=28.761758*u.deg,
                                         lon=-17.890659*u.deg,
                                         height=2200*u.m)
 
                 if evt_head['OBS_MODE'] in ('POINTING', 'WOBBLE'):
 
-                    alt_az_frame = AltAz(obstime=evt_time,
+                    alt_az_frame = AltAz(obstime=event_data['mjd'],
```

Comment:
I don't think you can pass a raw mjd value to `AltAz(obstime=)`, this needs to be a `Time` instance.

Reply:
Indeed, corrected.